src/sentry/templates/sentry/emails/incidents/trigger.html

```diff
@@ -71,6 +71,14 @@ <h5 style="margin-bottom: 12px;">Alert Rule Details</h5>
           <td>{{ alert_date_added }}</td>
         </tr>
       </table>
+
+      <p class="info-box">
+        {% if snooze_alert %}
+           <a class='mute' href="{{ snooze_alert_url }}">Mute this alert</a>
+        {% endif %}
+        This email was triggered by
+            <a href="{{ rule_link }}">{{ incident_name }}</a>
```

Comment:
Is `incident.title` the same as `incident.alert_rule.name`? 🤔 I think it is, but it's kinda weird that we store both separately.

Reply:
i think so, when testing it the incident_name matched the alert_rule name, not sure why we are storing both, but i think there is  some overlap